draft-rescorla-tls-ctls.md

```diff
@@ -116,8 +116,9 @@ a vector with a top range of a varint is denoted as:
      opaque foo<1..V>;
 ~~~~~
 
-With a few exceptions, cTLS replaces every integer in TLS
-with a varint.
+cTLS replaces all the integers in TLS (including code points) with
+varints; we do not show the structures which only change in this way.
```

Comment:
One thing I noticed when I tried to implement this is: If an implementation implements `opaque` as `vector<uint8>`, then the `uint8` entries might expand.  In general, it seems like there's no point to applying `varint` to `uint8`, since it only hurts.  Suggest:

```suggestion
cTLS uses the varint encoding for all multi-byte integers in TLS,
including:

* Values of type uint16, uint24, uint32, uint64
* Array and vector entries of these types
* Encoded lengths for vectors that allow more than 255 entries
* Enums that allow more than 255 entries

Values of type uint8, opaque values, and one-byte enums are not
affected.  We do not show the structures which only change in this way.
```